lua/entities/gmod_wire_rt_screen.lua

```diff
@@ -76,13 +76,17 @@ if SERVER then
         "Scroll X", "Scroll Y", "Scale X", "Scale Y"
     }
 
+	local screens = {}
```

Comment:
`local screens = screens or {}`

Reply:
This doesn't work for locals, if anything could be in the `WireLib` or `ENT` table

Reply:
It's fine as it is. Maybe it could be initialized to the return value of find by class so that it works through a script reload but that's minor

Reply:
> It's fine as it is. Maybe it could be initialized to the return value of find by class so that it works through a script reload but that's minor

I believe the file should only be reloaded if someone was editing that entity, which nobody would be doing on a live server so it shouldn't really matter too much.

Reply:
exactly it'd only affect wiremod developers so it doesn't matter